### PR TITLE
HP-655 - QA for additional PDAPS to heal aggregate config

### PR DESCRIPTION
**Pull request by mbkranz**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/HP-655

### Environments
QA

### Description of changes
QA of https://github.com/uc-cdis/cdis-manifest/pull/4062 (follow up to https://github.com/uc-cdis/cdis-manifest/pull/4051)

### CODE REVIEW

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-			"mds_url": "https://qa-heal.planx-pla.net",
-			"commons_url": "qa-heal.planx-pla.net"
+			"mds_url": "https://preprod.healdata.org",
+			"commons_url": "preprod.healdata.org"
+
 		},
 		"JCOIN": {
-			"mds_url": "https://qa-jcoin.planx-pla.net/",
-			"commons_url" : "qa-jcoin.planx-pla.net/",
+			"mds_url": "https://jcoin.datacommons.io/",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:9]:
```suggestion
			"mds_url": "https://qa-jcoin.planx-pla.net/",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {
 	"gen3_commons": {
 		"HEAL": {
-			"mds_url": "https://qa-heal.planx-pla.net",
-			"commons_url": "qa-heal.planx-pla.net"
+			"mds_url": "https://preprod.healdata.org",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:4]:
```suggestion
			"mds_url": "https://qa-heal.planx-pla.net",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+			"commons_url": "preprod.healdata.org"
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:5]:
```suggestion
			"commons_url": "qa-heal.planx-pla.net"
```

Comment (mbkranz) [reply qa-heal.planx-pla.net/metadata/aggregate_config.json]: @xritter1  I changed the mds_url of preprod back to qa-heal -- sorry about that!

**Review by xritter1 — APPROVED** [review]

**Pull request merged**